go/service/tlf.go

```diff
@@ -60,21 +60,14 @@ func (h *tlfHandler) CompleteAndCanonicalizeTlfName(ctx context.Context, arg key
 	// Append username in case it's not present. We don't need to check if it
 	// exists already since CryptKeys calls below transforms the TLF name into a
 	// canonical one.
-	arg.Query.TlfName = arg.Query.TlfName + "," + string(username)
+	arg.TlfName = string(username) + "," + arg.TlfName
```

Comment:
Change comment to Prepend.

Can you also mention that this makes username a writer on this TLF, which might be unexpected?